cla-backend-go/gerrits/repository.go

```diff
@@ -50,6 +52,55 @@ type repo struct {
 	dynamoDBClient *dynamodb.DynamoDB
 }
 
+func (repo *repo) ExistsByName(gerritName string) ([]*models.Gerrit, error) {
+	resultList := make([]*models.Gerrit, 0)
+	tableName := fmt.Sprintf("cla-%s-gerrit-instances", repo.stage)
+	filter := expression.Name("gerrit_name").Equal(expression.Value(gerritName))
+
+	expr, err := expression.NewBuilder().WithFilter(filter).Build()
+	if err != nil {
+		log.Warnf("error building expression for gerrit instances scan, error: %v", err)
+		return nil, err
+	}
+	// Assemble the query input parameters
+	scanInput := &dynamodb.ScanInput{
```

Comment:
Let's avoid scans as they are very slow and inefficient, and for this case, unnecessary. I have created a new index for the `gerrit_name` column.  I will push this new index/update to DEV environment along with the serverless permissions.

Update this to be a query and use the "gerrit-name-index" (see other `repository.go` examples which use an index).